### ISSUE TITLE
Support dotCover

### ISSUE THREAD

**Issue by csmacnz** [csmacnz/coveralls.net#7]
As a [Jetbrains dotcover](https://www.jetbrains.com/dotcover/) coverage user, I want to be able to publish my dotcover results to coveralls.io so that i can monitor my coverage results over time.

**Comment by StrangeWill** [comment]
Another option is to support the XML formats from [ReportGenerator](https://github.com/danielpalme/ReportGenerator) and let us convert from these various formats over to their XML format.

**Comment by csMACnz** [comment]
Yeah, that idea has occurred to me, since a lot of people already use ReportGenerator.
:+1:

**Comment by HarelM** [comment]
Has this been implemented in either way?

**Comment by csMACnz** [comment]
No it hasn't yet.

**Comment by HarelM** [comment]
Well, last time I contributed most of my work was discarded with the excuse of "this is not where I wanted to go" (which was a needed refactoring of duplicated code). This issue is now open more than two years.
It just doesn't feel right to contribute if this project is not getting any attention.
I feel like I should fork and create a different NuGet package but I think the users will suffer.
Also since there's not a lot of code here I would advise to convert it to .Net core so it will be able to run on Linux as well.
Let me know what you think and I'll see what I should do accordingly.

**Comment by csMACnz** [comment]
Fair comment. But that's the nature of open source I guess. You can fork and maintain your own copy, of course, but the original owner has the helm of the original, still.

I agree this needs to move to dotnet core, but it also needs to continue working where it currently works without requiring core. I have ideas of making this work.  Note that as of now there isn't a coverage solution (that I am aware of) for .Net (specifically dotnet core) that will actually run on Linux in a useful way, anyway. (This is something I am watching closely) And as I have tested this does already run on mono. I will move that to a new issue.

To the issue at hand, If you can provide a `ReportGeneratorParser` without refactoring the rest of the code It will be easier for me to accept and release it, If you have the time and inclination. Otherwise it will fall on me to do this, and you may be waiting a bit longer still.

I assume you are also are also a full time developer and will appreciate that the paid work comes before the open source, and that I do this project for fun, not profit.

Thanks for bearing with me, though. It is great to know people are using this, finding it useful and want to see it continue and develop.

**Comment by HarelM** [comment]
I had a very hard time adding the code last time due to unneeded complexity in the current code.
That's why I refactored it. I'm not planning on doing it again.
The refactoring I did took me about 2-3 hours - not something so time consuming, and left the code in a very good shape maintenance and extensiblity wise, unfortuntely you didn't like it.
I was hoping your response would be different, although I expected this is what you'd say.
You leave me no other choice than forking and creating another NuGet package which is disappointing.
I maintain an open source project which is a lot bigger and have a full time job as a team leader.
My 2 cents: 
If I'll get to a point where I don't have time to my open source project I'll try and make sure some is taking care of it and has the required permissions - this is not the case with this project IMHO.
Good luck!
